app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(author_id: params[:user_id], post_id: params[:post_id], **comment_params)
```

Comment:
@NicolasAndrehh great work on your create method, however, we are supposed to use the currently logged-in user (current_user) of the app using the current_user method we created instead of using user_id,
`hint`

```suggestion
   @comment = @current_user.comments.new(params)
   @comment.post_id = params[:post_id]
```